### DISCUSSION TITLE
Grouping API

### DISCUSSION THREAD

**Discussion by balaganski** [hkulekci/qdrant-php#42] [Q&A]
Hello,

Is it possible to utilize the Grouping API https://qdrant.tech/documentation/concepts/search/#grouping-api using this solution? 
I cannot find anything relevant in the source code...

Regards,
Alexei Balaganski

**Comment by hkulekci** [comment]
Hello @balaganski thanks for asking. For sure, we can implement it. I just started this client to build a solution for my hobby project. And I tried to improve it as much as I could. I have not used this grouping API before, so we have nothing related to grouping. I would be happy if anybody wanted to take this shot. If not, I will take a look in my free time.
